tests/test_result_to_dict.py

```diff
@@ -35,6 +35,14 @@ class Data(Base):
     data_enum = Column(Enum(MyEnum))
 
 
+def get_object(result):
+    info = result.get('object')
+    if info and isinstance(info, str):
+        dict_value = json.loads(info)
+        return dict_value
+    raise ValueError("Must be string")

```

Comment:
this code will raie exception also if `object` is not included in the data. 
what do you think about something similar to:

```
obj = result.get('object')
if obj: 
    if isinstance(obj, str):
         dict_value = json.loads(obj)
         return dict_value
    raise ValueError("Object values must be a type string")
```